src/Form/Control/Password.php

```diff
@@ -4,7 +4,45 @@
 
 namespace Atk4\Ui\Form\Control;
 
+use Atk4\Ui\Button;
+use Atk4\Ui\Js\JsExpression;
+
 class Password extends Line
 {
     public string $inputType = 'password';
+
+    /** By setting this variable to true you get an Eye-button on the right side which can toggle the visibility of the entered password */
+    public bool $eye;
```

Comment:
TODO for myself - should we use `false` or `null` by default

or `true` as default?

Reply:
I'm happy to put false as the default, this is from my point of view an option. 
But if there is a vote and true is selected as default, that's an easy change.

Reply:
My preference is to make this function enabled by default with as much as possible native look & feel.

I mean - not using `Input::$action` as someone else might want to use the right icon for regular input action.

Small not backgrounded icon will then look perfectly.

And the "slash" on the icon should be inverted - it should be based on the "actionable action", not on the current state.

Reply:
I do get where this is coming from, I'm not sure I know enough of css/js and atk4 to implement it though.
And yes, the icons was second pick because it's what is in Fomantic. 

I've found these on https://www.iconfinder.com/ which are free to use. they would be a better pick:
![eye_closed_light_icon](https://github.com/user-attachments/assets/f82ad8c1-407f-43bf-b4fe-5d9cda37ae52)
![eye_light_icon](https://github.com/user-attachments/assets/df48b65b-e566-49c0-aa32-ea3fff595d5e)

Reply:
We should stay with https://fomantic-ui.com/elements/icon.html#icon-set set only, you can open a feature request by them if the 1st icon is popular enough. For now, we should be fine with "eye slash" for unreveal.

Reply:
Works for me! :)

I'll do a request in the near future...